VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -96,4 +127,11 @@ class VideoSettings: NSObject
         aCoder.encodeObject(self.users, forKey: "users")
         aCoder.encodeObject(self.password, forKey: "password")
     }
+    
+    // MARK : String Methods
+    
+    func trim(text: String) -> String
+    {
+        return text.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceCharacterSet())
+    }
```

Comment:
Yeah I totally forgot that trim() from our own extension inside the app code. Nice catch